### ISSUE TITLE
Can't edit group (add user): "Could not validate group data"

### ISSUE THREAD

**Issue by SkaveRat** [passbolt/passbolt_api#247]
## Can't edit group (add user): "Could not validate group data"
* Passbolt Version: 2.0.1-debian (docker)
* Platform and Target:
-- Operating system: synology **docker**

### What you did
* Creating a group
* adding user
* add user to a group
* later on deleting this user
* create a new user with same name and mail address
* Trying to add this user to the group again causes above error message

### What happened
Decrypt/Encrypt message appears, (probably) aborts halfway through, showing a generic error message.  
Only message in logfile is "Could not validate group data"

### What you expected to happen
Adding the user to the group

**Comment by SkaveRat** [comment]
Some browser console logs:

![image](https://user-images.githubusercontent.com/570427/39056597-754a37d2-44b7-11e8-960b-b8d4db03e8c3.png)

**Comment by cedricalfonsi** [comment]
Hello @SkaveRat,

Did you upgrade your passbolt from a v1 ?

Can you execute the following command with the user who run your webserver :
`./bin/cake passbolt cleanup --dry-run`

**Comment by SkaveRat** [comment]
no, it's a clean v2 install.
comand output:

```
1 orphan records found in table Comments (soft deleted resources)                                                                            
20 orphan records found in table Secrets (soft deleted users)                                                                                
20 orphan records found in table Secrets (hard deleted permissions)                                                                          
41 issues detected, please run the same command without --dry-run to fix them.
```

**Comment by stripthis** [comment]
We can confirm that this is a bug. I'm working on a fix.
In the meantime you can run the cleanup for good and you will be able to add the user.
```
./bin/cake passbolt cleanup
```

**Comment by SkaveRat** [comment]
thanks. The cleanup fixed it

**Comment by stripthis** [comment]
Fixed with v2.0.4
